contrib/parseq-lambda-names/src/test/java/com/linkedin/parseq/lambda/TestMethodRef.java

```diff
@@ -0,0 +1,227 @@
+package com.linkedin.parseq.lambda;
+
+import java.util.Optional;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertTrue;
+
+
+public class TestMethodRef extends BaseTest {
+
+  private static final String CLASSNAME = TestMethodRef.class.getSimpleName();
+
+  TestMethodRef field = this;
+
+  @Test
+  public void testFunctionOnThis() {
+    Optional<String> description = getDescriptionForFunction(this::function);
+    assertTrue(description.isPresent());
+    assertNameMatch("function", "testFunctionOnThis", CLASSNAME, description.get());
+  }
+
+  TestMethodRef getTestMethodRef() {
+    return new TestMethodRef();
+  }
+
+  @Test
+  public void testFunctionOnThisChained() {
+    Optional<String> description = getTestMethodRef().getDescriptionForFunction(this::function);
+    assertTrue(description.isPresent());
+    assertNameMatch("function", "testFunctionOnThisChained", CLASSNAME, description.get());
```

Comment:
Can we prefix method references with "::" to more resemble how it looks like in source code e.g. "function" change to "::function"?